### DISCUSSION TITLE
Is Lorien might be one day an alternative to MS onenote?

### DISCUSSION THREAD

**Discussion by tharbad08** [mbrlabs/Lorien#112] [General]

**Comment by loirblitzt** [comment]
I feel like they should not be competing.
Onenote is a "keyboard" paper printable (vertical length), note taking software.
Lorien is a "tablet" infinite canvas note and·or scribble software.

As a user, i dont print my notes and i have a hard time expressing ideas with a keyboard. That is why I don't use OneNote.

I would reformulate your question into : what are the plans for the future ?
But the owner seems pretty buzy right now.

**Comment by tharbad08** [comment]
I don't need the print feature. Just keyboard and pen support.
I really want to find an OSS alternative to OneNote oneday.

**Comment by mbrlabs** [comment]
> I feel like they should not be competing

Yeah, i think so as well. With Lorien it's more about the simplicity of having an infinite (in all directions) canvas to draw/scribble on. No bells and whistles.

> But the owner seems pretty buzy right now

yep 😁

> I don't need the print feature. Just keyboard and pen support.

Drawing pens are already supported. Text fields maybe - but we have to wait until Godot 4 is out for that.

I'm moving this to the discussions sections, since it's not really an issue.
